types/trusted-types/index.d.ts

```diff
@@ -42,13 +43,30 @@ declare class TrustedTypePolicyFactory {
         policyOptions: Pick<TrustedTypePolicyOptions, Keys>,
         expose?: boolean,
         ): Pick<TrustedTypePolicy, 'name'|Keys>;
-    getExposedPolicy(name: string): TrustedTypePolicy|null;
     getPolicyNames(): string[];
 
     isHTML(value: any): value is TrustedHTML;
     isScript(value: any): value is TrustedScript;
     isScriptURL(value: any): value is TrustedScriptURL;
     isURL(value: any): value is TrustedURL;
+    getAttributeType(tagName: string, attrName: string, elemNs?: string, attrNs?: string): string | null;
```

Comment:
the style in the rest of the code is string|null (no space), please make consistent